app.js

```diff
@@ -4,7 +4,6 @@ const cardContainer = document.getElementById('books');
 
 class Book{constructor(title, author){ this.title = title; this.author = author }}
 
- 
```

Comment:
https://github.com/YannickZahinda/Awesome-book/blob/69dd6c82521046bdb291a274c0754740f0250498/app.js#L28-L32

- [ ] I can see that you don't display the author name when you add a new book, it should be 

```
       <h2 class="content">${storedBooks[i].title}, </h2>
       <h2 class="content">${storedBooks[i].author}, </h2> 
       <button class='remove' value="${storedBooks[i].title}" type="button"> Remove</button>
```

- [ OPTIONAL] How about emptying your input fields after adding the book look :point_down: 


||      Original     ||        Yours      ||
|-|----------------------|-| ----------------------|-|
||![555555555](https://user-images.githubusercontent.com/57562869/159468182-6ad5348a-77d6-4de5-b6c5-6d642d7dd33c.png)||![44444444444](https://user-images.githubusercontent.com/57562869/159468244-1a850346-0c69-4516-863e-5ef44607aff5.png)||